jwst/wfs_combine/wfs_combine.py

```diff
@@ -511,12 +511,12 @@ def interp_array(sci_data, dq_data, n_size):
         y = bad_dq[jj][0]
 
         # Linearly interpolate using scipy's griddata to fill in missing value
-        sci_data[y, x] = griddata(ga[:, 0:2], ga[:, 2], [(y, x)], method='linear')
+        sci_data[y, x] = griddata(ga[:, 0:2], ga[:, 2], [(y, x)], method='linear').item()
 
         # For those interpolations just done that result in a nan (because
         #    there may be too few pixels), check and redo with 'nearest'
         if np.isnan(sci_data[y, x]):
-            sci_data[y, x] = griddata(ga[:, 0:2], ga[:, 2], [(y, x)], method='nearest')
+            sci_data[y, x] = griddata(ga[:, 0:2], ga[:, 2], [(y, x)], method='nearest').item()
```

Comment:
I'm not sure about this one. Does it always return array of size 1?

Reply:
yes, it always returns a size 1 array.  Each of these calls to griddata interpolates `ga` at a single index `y, x` where a bad pixel is located.  `sci_data` is 2-D, and also indexed at `y, x`, but the call to `.item()` is necessary to fix "DeprecationWarning: Conversion of an array with ndim > 0 to a scalar is deprecated, and will error in future. Ensure you extract a single element from your array before performing this operation."

Reply:
and as an additional "safety", `.item()` without any arguments will raise an error if the array has size >1